augurpy/evaluate.py

```diff
@@ -88,28 +88,51 @@ def draw_subsample(
             while setting augur_mode = "permute" will generate a null distribution of AUCs for each cell type by
             permuting the labels
         subsample_size: number of cells to subsample randomly per type from each experimental condition
-        stratified: if `True` subsamples are stratified according to condition
+        categorical_data: if `True` subsamples are
         random_state: set numpy random seed and sampling seed
 
     Returns:
         Subsample of anndata object of size subsample_size
     """
     if augur_mode == "permut":
         # shuffle labels
+        adata = adata.copy()
```

Comment:
The reason I have to copy it is because Im changing the obs labels one line later, and it was changing them for the whole dataframe. I thought about only shuffling the labels of the already merged subsample adata object, but then the sampling would have been done based on the original labels ?

Reply:
No worries. It should be fine.